### PR TITLE
FT: Add backbeat route for putting delete marker

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: lib/routes/routeBackbeat.js [hunk 1]
```diff
@@ -229,11 +230,28 @@ function multipleBackendPutObject(request, response, bucketInfo, objMd, log,
...
+function multipleBackendDeleteObject(request, response, bucketInfo, objMd,
+    log, callback) {
+    const storageLocation = request.headers['x-scal-storage-class'];
+    const objectGetInfo = {
+        key: request.objectKey,
+        dataStoreName: storageLocation,
+    };
+    const reqUids = log.getSerializedUids();
+    return multipleBackendGateway.delete(objectGetInfo, reqUids, err => {
+        if (err) {
```
Comment (electrachong) [inline lib/routes/routeBackbeat.js]: do you want to add a log at this layer after all?

File: lib/routes/routeBackbeat.js [hunk 2]
```diff
@@ -229,11 +231,40 @@ function multipleBackendPutObject(request, response, bucketInfo, objMd, log,
...
+        return _respond(response, dataRetrievalInfo, log, callback);
+    });
+}
+
 const backbeatRoutes = {
     PUT: {
         data: putData,
         metadata: putMetadata,
         multiplebackendputobject: multipleBackendPutObject,
+        multiplebackenddeleteobject: multipleBackendDeleteObject,
```
Comment (philipyoo) [inline lib/routes/routeBackbeat.js]: I think this needs to be removed?

Comment (bennettbuchanan) [reply lib/routes/routeBackbeat.js]: Thanks, removed.

File: lib/routes/routeBackbeat.js (hunk 1 above)
Comment (bennettbuchanan) [reply lib/routes/routeBackbeat.js]: Updated.

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
E2E tests passed (CircleCi retried): http://ci.ironmann.io/gh/scality/Integration/16085#build-parameters.

**Pull request merged**